pkg/features/featureflags.go

```diff
@@ -0,0 +1,165 @@
+package features
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/golang/glog"
+)
+
+// Flags feature flags
+var Flags *Features
+var getVersion func() string
+
+func init() {
+	Flags = &Features{}
+	getVersion = getLinuxPTPPackageVersion
+}
+
+// Features ...
+type Features struct {
+	oc          bool
```

Comment:
it would be nice to have a short description for each feature. For example the feature title from Jira (not include Jira number).

Reply:
Would be useful to get people who are more familiar with the features to write the descriptions.